samples/suit/smp_transfer/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config APP_VERSION
+	int "Application version"
+	default 1
```

Comment:
This conflicts with an actual application version, which is set by having a `VERSION` file, you can't have a VERSION file because you want to change this per build but can you instead use `SUIT_ENVELOPE_SEQUENCE_NUM` and check against that rather than conflict with something that is already defined?